docs/pages/access-controls/device-trust/device-management.mdx

```diff
@@ -58,56 +55,31 @@ $ tsh device asset-tag
     To find the value chosen by Teleport, run the following command:
 
     ```code
-    $ tsh device collect
-    DeviceCollectedData {
-      ...
-      "serial_number": "(=devicetrust.asset_tag=)",
-      ...
-    }
+    $ tsh device asset-tag
+    (=devicetrust.asset_tag=)
     ```
   </TabItem>
 </Tabs>
 </Details>
-Use the serial number to register device:
-<Tabs>
```

Comment:
I've dropped the tabs here and instead made the `--os` value a variable, to call attention to what is actually the only distinction between the blocks.